package.json

```diff
@@ -50,15 +50,19 @@
     "react-helmet": "^3.1.0",
     "react-intl": "^2.1.5",
     "react-redux": "4.4.5",
-    "react-router": "^2.7.0",
-    "react-router-redux": "4.0.6",
     "redbox-react": "1.3.2",
     "redux": "^3.5.2",
+    "redux-little-router": "12.0.0",
     "redux-observable": "0.12.0",
     "request": "2.75.0",
     "rxjs": "^5.0.0-beta.11",
     "source-map-support": "0.4.3",
     "tough-cookie": "2.3.1",
     "url-search-params": "0.6.1"
+  },
+  "peerDependencies": {
```

Comment:
Ah okay, that totally makes sense. Peer dependencies seem like the more correct way to do it. Thanks for the explanation 👍